WordPress/src/main/res/layout/login_alert_http_auth.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="?attr/dialogPreferredPadding">
+
+    <android.support.design.widget.TextInputLayout
+        app:theme="@style/LoginTheme.EditText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <android.support.design.widget.TextInputEditText
```

Comment:
Made it a single line with 1c727bc. For context, reason I made a single line is to avoid having the dialog's Next button dropping out of view (hidden by the keyboard).